packages/PoppyPrint-Core.package/PPFormatter.class/class/formatString.class.noPattern.notifying.with..st

```diff
@@ -19,4 +19,10 @@ formatString: aString class: aClass noPattern: aBoolean notifying: anObject with
 		initForNode: methodNode;
 		config: aPPFormatterConfig.
 	methodNode accept: formatter.
-	^ aBoolean ifTrue: [self stripMethodPattern: formatter contents] ifFalse: [formatter contents]
\ No newline at end of file
+	
+	result := aBoolean
+		ifTrue: [self stripMethodPattern: formatter contents]
+		ifFalse: [formatter contents].
+	aPPFormatterConfig formatToMaxLineWidth ifTrue: [result := PPWindowFormatter new format: result withWindowWidth: aPPFormatterConfig maxLineWidth].
+	
+	^ result
```

Comment:
I guess this was used during debugging only? or is there a reason to keep the temp var around?

Reply:
no, this is actually needed for the LineWidth